File: backend/Entities/Models/Estado.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class Estado : BaseEntity
{
    public int Id { get; set; }

    public int IdPais { get; set; }

    public string Nome { get; set; } = null!;

    public string? Uf { get; set; }

    public string? CepInicial { get; set; }

    public string? CepFinal { get; set; }

    public virtual ICollection<FilialLastMile> FilialLastMiles { get; } = new List<FilialLastMile>();

    public virtual Pais IdPaisNavigation { get; set; } = null!;

    public virtual ICollection<TransportadorLastMile> TransportadorLastMiles { get; } = new List<TransportadorLastMile>();
}

File: frontend-with-menus/src/app/Business/Models/estado.ts
import { FilialLastMile } from "./FilialLastMile";
import { Pais } from "./pais";
import { TransportadorLastMile } from "./transportadorLastMile";

export interface Estado  {
    id: number;
    idPais: number;
    nome: string;
    uf: string | null;
    cepInicial: string | null;
    cepFinal: string | null;
    filialLastMiles: FilialLastMile[];
    idPaisNavigation: Pais;
    transportadorLastMiles: TransportadorLastMile[];
}